src/wms/src/test/java/org/geoserver/wms/WMSCascadeTestSupport.java

```diff
@@ -119,6 +121,54 @@ private void setupWMS130Layer() throws MalformedURLException, IOException {
                 new URL(mockPNGUrl), new MockHttpResponse(pngRoadsImage, "image/png"));
         wms13Client.expectGet(
                 new URL(mockJpegUrl), new MockHttpResponse(gifRoadsImage, "image/gif"));
+
+        // SET UP layer group of two cascaded wms layers
+        WMSLayerInfo group_lyr_1 = cb.buildWMSLayer("group_lyr_130");
```

Comment:
Think I've seen this already in other PRs... probably explains why you're getting conflicts.

Reply:
yes I used the same cascaded layer group for unit testing this implementation